LibZipSharp.props

```diff
@@ -3,6 +3,7 @@
     <_LibZipSharpNugetVersion>2.0.0</_LibZipSharpNugetVersion>
```

Comment:
Do we need to bump the version in this PR?

Reply:
Nope, 2.0.0 was never released, so it's fine